dm/dm-continuous-data-validation.md

```diff
@@ -0,0 +1,285 @@
+mysql-instances:
+  - source-id: "mysql1"
+    block-allow-list: "bw-rule-1"
+    validator-config-name: "global"
+validators:
+  global:
+    mode: full # "fast" is also allowed. "none" is the default mode, which means no validation is performed.
+    worker-count: 4 # The number of validation workers in the background. The default value is 4.
+    row-error-delay: 30m # If a row is not validated within the specified time, it will be marked as an error row. The default value is 30m, which means 30 minutes.
+```
+
+The configuration items are described as follows:
+
+* `mode`: validation mode. The possible values are `none`, `full`, and `fast`.
+    * `none`: the default value, which means no validation is performed.
+    * `full`: compares the changed row and the row obtained in the downstream database.
+    * `fast`: only checks if the changed row exists in the downstream database.
+* `worker-count`: the number of validation workers in the background. Each worker is a goroutine.
+* `row-error-delay`: if a row is not validated within the specified time, it will be marked as an error row. The default value is 30 minutes.
+
+For the complete configuration, refer to [DM Advanced Task Configuration File](/dm/task-configuration-file-full.md).
+
+### Method 2: Enable using dmctl
+
+To enable continuous data validation, run the `dmctl validation start` command:
+
+```
+Usage:
+  dmctl validation start [--all-task] [task-name] [flags]
+
+Flags:
+      --all-task            whether applied to all tasks
+  -h, --help                help for start
+      --mode string         specify the mode of validation: full (default), fast; this flag will be ignored if the validation task has been ever enabled but currently paused (default "full")
+      --start-time string   specify the start time of binlog for validation, e.g. '2021-10-21 00:01:00' or 2021-10-21T00:01:00
+```
+
+* `--mode`: specify the validation mode. The possible values are `fast` and `full`.
+* `--start-time`: specify the start time for validation. The format follows `2021-10-21 00:01:00` or `2021-10-21T00:01:00`.
+* `task`: specify the name of the task to enable continuous validation for. You can use `--all-task` to enable validation for all tasks.
+
+For example:
+
+```shell
+dmctl --master-addr=127.0.0.1:8261 validation start --start-time 2021-10-21T00:01:00 --mode full my_dm_task
+```
+
+## Use continuous data validation
+
+When you use continuous data validation, you can use dmctl to view the status of the validation and to handle the error rows. "Error rows" refers to the rows that are found to be inconsistent between the upstream and downstream databases.
+
+### View the validation status
+
+You can view the validation status using either of the following methods:
+
+Method 1: run the `dmctl query-status <task-name>` command. If continuous data validation is enabled, the validation result is displayed in the `validation` field of each subtask. Example output:
+
+```json
```

Comment:
I remember that JSON doesn't allow comments in its body. It seems the comments is corruptively rendered with unexpected red background😥 

Maybe try JS instead.

Reply:
The comments look good on our website. We might blame the rendering on GitHub. 😉 
<img width="827" alt="image" src="https://user-images.githubusercontent.com/59654584/183824560-a832d79d-be7a-42c8-817a-2d49e870f150.png">
https://netlify--preview-docs.netlify.app/zh/tidb/dev/dm-continuous-data-validation